gcp/modules/helm-initializer/main.tf

```diff
@@ -15,3 +15,11 @@ module "system_tiller" {
   secrets_dir      = "${var.secrets_dir}"
   tiller_namespace = "${var.tiller_namespace}"
 }
+
+resource "null_resource" "wait_for_tiller" {
```

Comment:
Please add a comment explaining this, i.e. referencing the confusing "context deadline exceeded" error message.

Reply:
Context deadline appeared to be unrelated issue, however, we still this because of "could not find a ready tiller pod" issue.

Reply:
Cool.